src/cloud/enrichments/README.md

```diff
@@ -377,6 +418,24 @@ This file contains gaze data in world camera coordinates. For a definition of th
 | **gaze x [px]** | Float value representing the x-coordinate of the mapped gaze point in world camera pixel coordinates.
 | **gaze y [px]** | Same as "gaze x [px]" but for the y-coordinate.     |
 | **worn** | This value indicates whether the Pupil Invisible Glasses have been worn by a subject at this point in time. `1.0` indicates that it has been worn, while `0.0` indicates that is has not been. Added in version 2 of this enrichment.    |
+| **fixation id** | If this gaze samples belongs to a fixation event, this is the corresponding id of the fixation. Otherwise this field is empty.     |
+
+
+#### fixations.csv
+This file contains fixations detected in the gaze data stream.
+The corresponding gaze samples that belong to each fixation can be determined from the `gaze.csv` file using the `fixation id` fields.
+
+
+| Field | Description | 
+| -------- | -------- | 
+| **section id** | Unique identifier of the corresponding section.     |
+| **recording id** | Unique identifier of the recording this sample belongs to.     |
+| **fixation id** | Identifier of the fixation. The counter starts at the beginning of the recording.     |
+| **start timestamp [ns]** | UTC timestamp in nanoseconds of the start of the fixation.     |
```

Comment:
The start of the fixation can be before the section start, which value do we want to provide here?

Eg.
```

time            1  2  3  4  5  6  7  8  9 10 11
fixation id           1  1  1  1  2  2  2  2  2
section                     [           ]
```

Here the section is from t=5 to t=9, and there are two fixations (f1 and f2) in that range.

Do we want to output the start for f1 as t3 when it actually started or t5 which is when it was first occurred within the section.

Reply:
Regarding the starting and end points, I think we should use the actual timestamps even if they are outside of the section. Otherwise, this could impact fixation duration statistics. We could disclaim this in the docs.

What happens then is that we are not able to calculate the mean location on surface "correctly" because we are not able to map those gaze points to the surface that are outside of the section. I guess then we just do best effort and calculate the location based on what could be mapped.

I don't think this decision matters too much as the average section should contain a ton of fixations and having 2 of them being odd is fine. It is easier to cut down the fixation if it is reported in full duration than the other way around though, in case it does matter, so that seems like the better trade-off to me.